ml_ops.sh

```diff
@@ -74,13 +74,13 @@ hdfs dfs -rm -R -f ${HDFS_SCORED_CONNECTS}
 
 # Add -p <command> to execute pre MPI command.
 # Pre MPI command can be configured in /etc/duxbay.conf
-# In this script, after the line after --mpicmd ${MPI_CMD} add:
-# --mpiprep ${MPI_PREP_CMD}
+# In this script, after the line after -c ${MPI_CMD} add:
```

Comment:
Didn't find a quick solution for empty parameters. I will keep looking, I hope this commit make it to dev anyway.